soccer/Context.hpp

```diff
@@ -35,4 +35,6 @@ struct Context {
 
     std::optional<QPointF> ball_command;
     std::optional<Geometry2d::TransformMatrix> screen_to_world_command;
+
+    bool is_simulation;
```

Comment:
@kylestach should this be an optional or is a bool fine

Reply:
Boolean is fine. We are always either in sim or not in sim.